.github/actions/build-docker/action.yml

```
@@ -75,7 +68,7 @@ runs:
         # Hard coding our dockerhub imnage name
```

While you're at it...

```suggestion
        # Hard coding our dockerhub image name
```